cinema/views.py

```diff
@@ -0,0 +1,41 @@
+from django.http import HttpRequest
+from django.shortcuts import get_object_or_404
+from rest_framework import status
+from rest_framework.response import Response
+from rest_framework.decorators import api_view
+
+from .models import Movie
+from .serializers import MovieSerializer
+
+
+@api_view(["GET", "POST"])
+def movies(request: HttpRequest) -> Response:
+    if request.method == "GET":
+        movies = Movie.objects.all()
+        serializer = MovieSerializer(movies, many=True)
+        return Response(serializer.data, status=status.HTTP_200_OK)
+
+    if request.method == "POST":
```

Comment:
no need for this condition here, you can just unindent below code because only two methods are accepted and you have return statement above